MonitorControl.xcodeproj/project.pbxproj

```diff
@@ -362,6 +368,8 @@
 				ja,
 				pl,
 				hu,
+				tr,
+				"es-419",
```

Comment:
Is es-419 a specific variant of Spanish? Because when I add Spanish it just uses `es`

Reply:
It's Latin American Spanish (México, Venezuela, etc)
The Spain Spanish (sorry for the comparison) is only `es`